trestle/utils/fs.py

```diff
@@ -161,7 +161,8 @@ def get_contextual_model_type(path: pathlib.Path = None) -> Tuple[Type[OscalBase
     return model_type, full_alias
 
 
-def get_stripped_contextual_model(path: pathlib.Path = None) -> Tuple[Type[OscalBaseModel], str]:
+def get_stripped_contextual_model(path: pathlib.Path = None,
```

Comment:
I don't know what the 'none' is doing here. I think given what I see below in like 173 we should just force people to pass it. Optional arguments for no good reason are a pain (and the type annotation needs to be wrapped in optional.

Reply:
this is getting the model type. The output tuple was correct if it is passing a class and not a class instance.

Reply:
I believe the None was by design - You can get contextual model without passing a path - in that case cwd is used.
The return types I agree - got taken out in git merge - added back.